hack/devtools/rp_dev_helper.sh

```diff
@@ -1,39 +1,471 @@
+        "ubi8/nodejs-18" "ubi8/go-toolset" "app-sre/managed-upgrade-operator" "app-sre/hive" "distroless/genevamdm" "distroless/genevamdsd"
+        "aro" "fluentbit")
+
+check_acr_repos() {
+    err_str="Usage: $0 <RESOURCE_GROUP> [GIT_COMMIT] [SKIP_DEPLOYMENTS]. Please try again"
+    local resource_group=${1?$err_str}
+    local git_commit=${2:-"no-commit"}
+    local skip_deployments=${3:-"true"}
+
+    # Don't skip deployment creation when skip_deployments was set to 'false'
+    if is_it_boolean $skip_deployments && ! $skip_deployments; then
+        log "'skip_deployments' was set to 'false'. ❌⏩ Don't skip acr repo mirroring in ResourceGroup $resource_group"
+        return 1
+    fi
+   
+    # check_jq_installed - Might not needed
+    # Get the first Azure Container Registry (ACR) name under ResourceGroup
+    acr_name="$(az acr list --resource-group $resource_group | jq -r '.[0].name')"
+    if [[ -z "$acr_name" ]]; then
+        abort "🔴❌✈️ Error: There are no Azure Container Registries under ResourceGroup '$resource_group'."
+    fi
+    
+    # Do all the needed repos already imported
+    local -a missing_repos_names=()
+    for repo in ${acr_repos[@]}; do
+        if [[ ${repo} == "aro" && $git_commit != "no-commit" ]]; then
+            if ! check_acr_repo $acr_name $repo $skip_deployments $git_commit; then
+                missing_repos_names+=("$repo")
+            fi
+        else
+            if ! check_acr_repo $acr_name $repo $skip_deployments; then
+                missing_repos_names+=("$repo")
+            fi
+        fi
+    done
+    if [ ${#missing_repos_names[@]} -eq 0 ]; then
+        log "🟢✈️ All repositories exist in ACR '$acr_name'."
+        return 0
+    fi
+    echo -e "🔴✈️ Some repositories are missing and need to be imported.\nRepositories: ${missing_repos_names[@]}\n"
+    return 1
+}
+
+import_geneva_image() {
+    err_str="Usage: $0 <REPOSITORY> <Tag> <DST_ACR_NAME>. Please try again"
+    local repository=${1?$err_str}
+    local tag=${2?$err_str}
+    local dst_acr_name=${3?$err_str}
+    if ! check_acr_repo ${dst_acr_name} ${repository} "true" ${tag} ;then
+        az acr import --name ${dst_acr_name}.azurecr.io/${repository}:${tag} --source linuxgeneva-microsoft.azurecr.io/${repository}:${tag}
+        log "🟢✈️ Imported repository '${repository}' to ACR '${dst_acr_name}'"
+    else
+        log "⏭️📦 Skip importing repository '${repository}' to ACR '${dst_acr_name}', since it already exist with tag ${tag}. Import can not run twice"
+    fi
+}
+
+check_keyvault_certificate() {
+    err_str="Usage $0 <KEYVAULT> <CERTIFICATE> [SKIP_DEPLOYMENTS]. Please try again"
+    local key_vault=${1?$err_str}
+    local certificate=${2?$err_str}
+    local skip_deployments=${3:-"true"}
+
+    # Don't skip deployment creation when skip_deployments was set to 'false'
+    if is_it_boolean $skip_deployments && ! $skip_deployments; then
```

Comment:
Double quote to prevent word splitting.

```suggestion
    local key_vault="${1?$err_str}"
    local certificate="${2?$err_str}"
    local skip_deployments="${3:-"true"}"

    # Don't skip deployment creation when skip_deployments was set to 'false'
    if is_it_boolean "$skip_deployments" && ! $skip_deployments; then
```